assets/css/src/amp-admin.css

```diff
@@ -6,6 +6,17 @@
 	position: relative;
 }
 
+.theme-browser .theme .dashicons-external,
+.plugin-action-buttons .dashicons-external {
+	vertical-align: text-bottom;
+	margin-left: 5px;
+}
+
+.theme-browser .theme .theme-screenshot img {
+	height: 100%;
+	object-fit: contain;
+}
```

Comment:
This is needed because the theme screenshots from amp-wp.org are not the same aspect ratio as those from WordPress.org:

Before | After
-------|-------
![image](https://user-images.githubusercontent.com/134745/142031525-b84bd678-0ae8-4295-ba19-60315f395e16.png) | ![image](https://user-images.githubusercontent.com/134745/142031456-a2a30c3f-d699-43cc-9da3-047efbfce938.png)